components/ActionSheet/src/private/MDCActionSheetItemTableViewCell.m

```diff
@@ -0,0 +1,215 @@
+                                                          constant:leadingConstant];
+  _leadingTitleConstraint.active = YES;
+  CGFloat width = CGRectGetWidth(self.contentView.frame) - (kLeadingPadding * 2);
+  [NSLayoutConstraint constraintWithItem:_textLabel
+                               attribute:NSLayoutAttributeWidth
+                               relatedBy:NSLayoutRelationEqual
+                                  toItem:nil
+                               attribute:NSLayoutAttributeNotAnAttribute
+                              multiplier:1
+                                constant:width].active = YES;
+
+  if (!_inkTouchController) {
+    _inkTouchController = [[MDCInkTouchController alloc] initWithView:self];
+    [_inkTouchController addInkView];
+  }
+
+  _imageView = [[UIImageView alloc] init];
+  [self.contentView addSubview:_imageView];
+  _imageView.translatesAutoresizingMaskIntoConstraints = NO;
+  _imageView.alpha = kImageAlpha;
+  [NSLayoutConstraint constraintWithItem:_imageView
+                               attribute:NSLayoutAttributeTop
+                               relatedBy:NSLayoutRelationEqual
+                                  toItem:self.contentView
+                               attribute:NSLayoutAttributeTop
+                              multiplier:1
+                                constant:16.f].active = YES;
+  [NSLayoutConstraint constraintWithItem:_imageView
+                               attribute:NSLayoutAttributeLeading
+                               relatedBy:NSLayoutRelationEqual
+                                  toItem:self.contentView
+                               attribute:NSLayoutAttributeLeading
+                              multiplier:1
+                                constant:16.f].active = YES;
+  [NSLayoutConstraint constraintWithItem:_imageView
+                               attribute:NSLayoutAttributeWidth
+                               relatedBy:NSLayoutRelationEqual
+                                  toItem:nil
+                               attribute:NSLayoutAttributeNotAnAttribute
+                              multiplier:1
+                                constant:24.f].active = YES;
+  [NSLayoutConstraint constraintWithItem:_imageView
+                               attribute:NSLayoutAttributeHeight
+                               relatedBy:NSLayoutRelationEqual
+                                  toItem:nil
+                               attribute:NSLayoutAttributeNotAnAttribute
+                              multiplier:1
+                                constant:24.f].active = YES;
+  _widthConstraint.constant = CGRectGetWidth(self.contentView.frame) - (kLeadingPadding * 2);
+}
+
+- (void)layoutSubviews {
+  [super layoutSubviews];
+
+  NSMutableParagraphStyle *paragraphStyle = [[NSMutableParagraphStyle alloc] init];
+  paragraphStyle.hyphenationFactor = 1.f;
+  NSDictionary<NSString *, id> *attributes = @{ NSParagraphStyleAttributeName : paragraphStyle };
```

Comment:
This seems like a lot of complexity just to set the hyphenationFactor.
Can we drop that requirement?

Reply:
Removed